client/src/components/UserForm.jsx

```diff
@@ -20,7 +20,14 @@ class UserForm extends React.Component {
   }
 
   handleClick(event) {
-    // model from Joe
+    event.preventDefault();
+    let newUserObject = {
+      name: this.state.username,
+      income: Number(this.state.income),
```

Comment:
Shouldn't this be `this.state.income * 100`?

Reply:
Agreed